app/models/post.rb

```diff
@@ -0,0 +1,14 @@
+class Post < ApplicationRecord
+  belongs_to :user
+  has_many :comments, foreign_key: 'author_id'
+  has_many :likes, class_name: 'Like', foreign_key: 'post_id', dependent: :destroy
```

Comment:
- [ ] In the user model you named the foreign key Author_id therefore you should mention this in the belongs to part of the relation as well like below:
```
belongs_to :author, class_name: 'User'
```